config/initializers/disable_xml_params.rb

```diff
@@ -1,3 +1,3 @@
 # Protect against injection attacks
 # http://www.kb.cert.org/vuls/id/380039
-ActionDispatch::ParamsParser::DEFAULT_PARSERS.delete(Mime::XML)
+# ActionDispatch::ParamsParser::DEFAULT_PARSERS.delete(Mime::XML)
```

Comment:
Why is this disabled? If necessary: remove file.

Reply:
I think `rails app:update` disabled this.  This was related to a security issue(CVE-2013-0156) in rails <= 3 and so isn't needed anymore in newer rails version. I remove the file.